src/components/Select/Select.tsx

```diff
@@ -130,6 +137,12 @@ export const Select: ChakraComponent<
         isInvalid ? "dark.ui.error.primary" : "dark.ui.typography.body"
       );
 
+      if (onChange && defaultValue) {
+        console.warn(
+          "NYPL Reservoir Select: Both an `onChange` prop (used for controlled components) and a `defaultValue` prop (used for controlled components) were passed. `defaultValue` will be ignored."
```

Comment:
1. Combine the text in parenthesis so it's only used once "(both used for controlled components)..."
2. Add a tests for this.

Reply:
1. onChange is for controlled, and defaultValue is for uncontrolled. The warning is because they shouldn't be used together, unless I am misunderstanding. 2. Will add a test.

Reply:
Ah yes, that's right for 1 but then the text in parenthesis should say "uncontrolled" for `defaultValue`.

Reply:
Ah! Good catch. Thanks!